docs/rules/no-restricted-browser-globals-during-ssr.md

```diff
@@ -0,0 +1,40 @@
+# Disallow access to global browser APIs during SSR (`lwc/no-restricted-browser-globals-during-ssr`)
+
+Browser APIs must not be accessed when SSR is being done. This rule prevents usage of browser APIs like `DOMParser`, `DocumentFragment` etc.
+in `connectedCallback` (and in methods called from `connectedCallback` or anywhere when SSR is being done).
+
+## Rule Details
+
+Examples of **incorrect** code for this rule:
+
+```js
+import { LightningElement } from 'lwc';
+
+export default class Foo extends LightningElement {
+    connectedCallback() {
+        const parser = new DOMParser();
+    }
+}
+```
+
+Examples of **correct** code for this rule:
+
+```js
+import { LightningElement } from 'lwc';
+
+export default class Foo extends LightningElement {
+    renderedCallback() {
+        const parser = new DOMParser();
+    }
+}
+```
+
+## Options
+
+The rule takes one option, an object, which has one key `restricted-globals` which is an object. The keys in the object
```

Comment:
I would add a note here stating that by default, all methods and interfaces exposed in Node.js are allow-listed by default.

Reply:
Done.